recommend/yaml/rules.yaml

```diff
@@ -115,7 +115,7 @@ policyRules:
       never be used in prod env, or if used, should be used only in certain time frames.
       Examples include, dynamic package management tools, mii-tool, iptables etc
   rules:
-    pathRule:
+    fileRule:
```

Comment:
Can you please add an example processRule in the rules yaml? Thanks

Reply:
@nyrahul Done - eca7114